governance/init-governance.php

```diff
@@ -120,7 +120,13 @@ private static function load_governance_configuration() {
 				$block_settings_for_user   = $governance_rules_for_user['blockSettings'];
 				$nested_settings_and_css   = NestedGovernanceProcessing::get_nested_settings_and_css( $block_settings_for_user );
 				BlockLocking::init( $governance_rules_for_user['allowedFeatures'] );
-				Analytics::record_usage();
+
+				$epoch_time = floor( microtime( true ) );
+
+				if ( ( $epoch_time % 10 ) === 0 ) {
+					// Sample results. Only send analytics on 10% of configuration loads.
```

Comment:
So `microtime` returns seconds, so this will send when  the editor is loaded and it happens to be 0, 10, 20, etc.

I'm a little more concerned about this undercounting than the Block Data API, which when in high use is called almost constantly. 

But we can check it out. The median number of MC stat hits for October has 348 so we would expect it to be 35 going forward, that still leaves enough room to notice changes.

But we should test that assumption after launch to see if the stats drop by more than 90 percent.